frontend/src/views/overbooking-number.js

```diff
@@ -312,19 +313,25 @@ function OverbookingNumber() {
               styles={{
                 margin: "auto",
                 marginTop: 0,
-                marginBottom: 10
+                marginBottom: 16,
+                width: 64,
+                height: 64,
+                backgroundColor: "#F4F6FA",
+                "@media (min-width: 375px)": {
+                  width: 64,
```

Comment:
Why do both the normal view and the media query have the same width and height?

Reply:
It is necessary because PictureBoss componente have a media query and I want to overwrite